src/components/MaplibreMap.jsx

```diff
@@ -237,47 +244,45 @@ const MaplibreMap = ({ view, setView }) => {
     const { features } = event
 
     if (features && features.length > 0) {
-
-      const feature = features[0];
+      const feature = features[0]
       if (['sites-cluster-layer', 'sites-unclustered-layer'].includes(feature.layer.id)) {
-        mapRef.current.getCanvas().style.cursor = '';
+        mapRef.current.getCanvas().style.cursor = ''
       }
     }
   }
 
-
   const MapAndTableControlsWrapper = () => {
     const map = mapRef.current
-    
+
     if (map) {
-      return <MapAndTableControls map={map.getMap()} view={view} setView={setView} isLeafletMap={false}/>
+      return <MapAndTableControls map={map.getMap()} view={view} setView={setView} />
     }
   }
 
   const handleMapLoad = () => {
     if (mapRef.current) {
       const map = mapRef.current.getMap()
-      const customImage = new Image();
+      const customImage = new Image()
       customImage.onload = () => {
         if (!map.hasImage('custom-icon')) {
-          mapRef.current.addImage('custom-icon', customImage, { sdf: false });
+          mapRef.current.addImage('custom-icon', customImage, { sdf: false })
         }
       }
-      customImage.src = customIcon; 
+      customImage.src = customIcon
     }
   }
 
   return (
     <>
       <Map
         ref={mapRef}
-        style={{width: '100%', height: '100%'}}
+        style={{ width: '100%', height: '100%' }}
         initialViewState={{
-          longitude: mapCenter && mapCenter[1] || defaultLon,
-          latitude: mapCenter && mapCenter[0] || defaultLat,
-          zoom: mapZoom || defaultMapZoom
+          longitude: (mapCenter && mapCenter[1]) || defaultLon,
+          latitude: (mapCenter && mapCenter[0]) || defaultLat,
+          zoom: mapZoom || defaultMapZoom,
         }}
-        mapStyle='https://demotiles.maplibre.org/style.json'
+        mapStyle="https://demotiles.maplibre.org/style.json"
```

Comment:
A style is required here so we can later access fonts for the cluster count text.
However, including it here makes it render the vector tiles on the map before the world imagery layer is rendered on top of it.
Can you figure out a way to keep the this as the map style but to hide the vector tiles before they are displayed on the map? This was on my list of potential improvements and something I discussed with @saanobhaai